src/main/java/org/apache/maven/plugins/shade/mojo/ShadeMojo.java

```diff
@@ -762,55 +766,72 @@ private File resolveArtifactSources( Artifact artifact )
 
     private List<ResourceTransformer> getResourceTransformers()
     {
+        final List<ResourceTransformer> resourceTransformers = new LinkedList<>( getDefaultResourceTransformers() );
         if ( transformers == null )
         {
-            return Collections.emptyList();
+            return resourceTransformers;
         }
+        resourceTransformers.addAll( Arrays.asList( transformers ) );
+        return resourceTransformers;
+    }
 
-        return Arrays.asList( transformers );
+    private List<ResourceTransformer> getDefaultResourceTransformers()
+    {
```

Comment:
Since we're not merging, so you can return a list with these 2 transformers.
Also document this on the `transfomers`-parameter.

Reply:
guess it is addressed by previous comment, we want to merge ;)